common_examples/tcp_client/main/tcp_client.c

```diff
@@ -1,8 +1,3 @@
-/*
- * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
- *
- * SPDX-License-Identifier: Unlicense OR CC0-1.0
- */
```

Comment:
why did you remove the copyright header?

Reply:
By mistake :smiley:  The history of the update is as follows: I need to update LAN867x TCP example at first because it didn't perform well. Then I replaced the `common` one with `LAN867x` which didn't contain the copyright. Luckily such mistakes will be prevented in the future since a new pre-commit hooks were established on this repo in the meantime.

Reply:
Added back but license changed to Apache-2.0 since `Unlicense OR CC0-1.0` is no more allowed.

Reply:
I think you can still use the `Unlicense` in examples, just need to configure it in

https://github.com/espressif/esp-eth-drivers/blob/b1323107c396c8e910b5761d9ce2c89b7bed24b5/.ci/check_copyright_config.yaml#L9

(but it's up to you, LGTM from my side)